plugins/generator-1.18.2/forge-1.18.2/templates/elementinits/villagertrades.java.ftl

```diff
@@ -41,24 +41,46 @@ import net.minecraft.world.entity.npc.VillagerTrades;
 
 @Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE) public class ${JavaModName}Trades {
 
+    <#if trade.hasWanderingTrade()>
+    @SubscribeEvent public static void registerWanderingTrades(WandererTradesEvent event) {
+        <#list villagertrades as trade>
+            <#list trade.tradeEntries as tradeEntry>
+                <#if tradeEntry.villagerProfession == "WanderingTrader">
+                    <#list tradeEntry.entries as entry>
+                        event.getGenericTrades().add(
+                        new BasicItemListing(
+                        ${mappedMCItemToItemStackCode(entry.price1, entry.countPrice1)},
+                        <#if !entry.price2.isEmpty()>${mappedMCItemToItemStackCode(entry.price2, entry.countPrice2)},</#if>
+                        ${mappedMCItemToItemStackCode(entry.offer, entry.countOffer)},
+                        ${entry.maxTrades}, ${entry.xp}, ${entry.priceMultiplier}f
+                        ));
+                    </#list>
+                </#if>
+            </#list>
+        </#list>
+    }
+    </#if>
+
+    <#if trade.hasNormalTrade()>
```

Comment:
```suggestion
    <#if w.hasVillagerTrades(false)>
```

Reply:
Thanks!

Reply:
Seems like this doesn't work, it causes an error